watertap/core/util/initialization.py

```diff
@@ -151,7 +151,7 @@ def interval_initializer(
         feasibility_tol : tolerance to use for FBBT (default: 1e-6)
         default_initial_value: set uninitialized variables to this value (default: 0.0)
         logger : logger to use (default: watertap.core.util.initialization)
-
+        fail_flag: boolean that raises exception if True or logs warning when an exception occurs
```

Comment:
this docstring is a bit cryptic imo... why not use the same dosctring for `fail_flag` as in the other functions in the file?

```
fail_flag : Boolean argument to specify error or warning
(Default: fail_flag=False produces logger warning. Set fail_flag=True to raise an error and stop
the initialization routine.)
```

Reply:
good point

Reply:
Should False really be the default? Imo, most users are unlikely to even know that these config options exist, so we should assume the default will take effect more often than not. So I think it's better to make the user manually relax the error if they're confident that they've encountered a "select case where InfeasibleConstraintException will be raised" but the model can still get to the right point. If False, is the default, it allows unaware users or users that have "warning fatigue" (e.g. there are some flowsheets that display a lot of warnings, so this one may get lost in the mess) to ignore the warning when it ought to be addressed